### DISCUSSION TITLE
Bar is gone and only shows when I type polybar in the terminal.

### DISCUSSION THREAD

**Discussion by LilEggrolls** [polybar/polybar#3003] [Q&A]
I was watching a guide on how to customise polybar when eventuallt the bar vanished... I retraced mt steps and change everything back to default, but the bar is still gone. When I type polybar into the terminal it pops up but only while the terminal is still open. I'm not able to chech for logs because everytime I do polybar -l get,  a error: Uncaught exception, shutting down: Missing argument for option -l. Any help at all would be nice.

**Comment by h0r4ci0** [comment]
That's because you're running the polybar like a process inside of the terminal. If you kill it, you kill polybar too.

You can use this script to run polybar out of the terminal.

```sh
#!/usr/bin/env bash

# Terminate already running bar instances
killall -q polybar
# If all your bars have ipc enabled, you can also use 
# polybar-msg cmd quit

# Launch bar1 and bar2
echo "---" | tee -a /tmp/polybar1.log /tmp/polybar2.log
#polybar black >>/tmp/polybar1.log 2>&1 & disown
polybar example -r >>/tmp/polybar1.log 2>&1 & disown
echo "Bars launched..."
```
